scripts/train.py

```diff
@@ -1,8 +1,10 @@
 import argparse
```

Comment:
We probably want a docstring here saying in a few words what this script does (and outputs) and with what kind of input data.

Reply:
Also an example of how it should be called or used.

Reply:
resolved in 778aee4